### ISSUE TITLE
Element with aria-hidden has no focusable content - Passed Example 3 is a fail

### ISSUE THREAD

**Issue by dd8** [act-rules/act-rules.github.io#1386]
Passed example 3 assumes that all screen reader users access content via the keyboard and can't see the content on screen:
https://act-rules.github.io/rules/6cfa84#passed-example-3

The button is still focusable by clicking or touch - add a button:focus{ border: 1px solid red; } CSS rule in dev tools to see this

This example would cause problems for the following people:

- dyslexics with perfect vision who need a screen reader to voice text on screen
- people with low vision who may be able to see outlines but can't read text

In both cases they would see the button, and could focus it by clicking or touch, but it wouldn't voice anything

--- or ---

If authoring a proposal for a new rule, choose from one of the issue templates:

- [Atomic Rule Template](https://raw.githubusercontent.com/act-rules/act-rules.github.io/develop/pages/design/atomic-template-empty.md)
- [Composite Rule Template](https://raw.githubusercontent.com/act-rules/act-rules.github.io/develop/pages/design/composite-template-empty.md)

**Comment by WilcoFiers** [comment]
OK we didn't get to this in the call, but I think you're right. The way this should have been done is to disable the button. I'll update the rule.

**Comment by WilcoFiers** [comment]
Discussed this on the call today. i'm going to try to fix this by checking if the element can be focused by touch or click. Need to figure out how to define that, but I think that's a good addition to this rule. Thanks for the suggestion Mark!
